Gemfile

```diff
@@ -4,7 +4,6 @@ git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 # Specify your gem's dependencies in rails_async_methods.gemspec.
 gemspec
 
-gem "pg"
```

Comment:
Bundler is complaining about pg being included twice as it is also defined as dependency in gemspec